roles/aap_setup_download/defaults/main.yml

```diff
@@ -9,6 +9,9 @@
 # which version of the installer to download
 aap_setup_down_version: "2.1"
 
+# Which RHEL version are you using (8 or 9)
+aap_setup_rhel_version: "{{ ansible_distribution_major_version | default(8, true) }}"
```

Comment:
Should the default here be 9 instead of 8 (to match what the readme change states)?

Reply:
I thought of that when reading it, I was not sure if the meaning was latest for that version and the major version default would be 8 for now. I am not sure we should push 9 until at least 9.1 or 9.2.

Reply:
Yes the default should be 8 I think still. The readme doesn't specify which one is default but I can add that in